meta/.phpstorm.meta.php

```diff
@@ -81,6 +81,16 @@ function elementType($argNum) {
   override(\array_values(0), type(0));
   override(\array_combine(0), type(1));
 
+  override(\DOMDocument::importNode(0), type(0));
+  override(\DOMNode::appendChild(0), type(0));
+  override(\DOMNode::insertBefore(0), type(0));
+  override(\DOMNode::cloneNode(0), type(0));
```

Comment:
`DOMNode::cloneNode` accepts boolean as first parameter, not `DOMNode`

Reply:
I don't know how this got here. Fixed